services/core-web/src/components/navigation/NotificationDrawer.tsx

```diff
@@ -246,19 +243,4 @@ const NotificationDrawer: FC<INotificationDrawerProps> = (props) => {
   );
 };
 
-const mapStateToProps = (state) => ({
-  userInfo: getUserInfo(state),
-  activities: getActivities(state),
-});
-
-const mapDispatchToProps = (dispatch) =>
-  bindActionCreators(
-    {
-      fetchActivities,
-      markActivitiesAsRead,
-      storeActivities,
-    },
-    dispatch
-  );
-
-export default connect(mapStateToProps, mapDispatchToProps)(NotificationDrawer);
+export default NotificationDrawer;
```

Comment:
Yeah!  Get outta here mapStateToProps and mapDispatchToProps!

Reply:
Thanks again for talking it through with me when I was too focused on the infinite loop I was getting!